.github/actions/cmx-versions/index.js

```diff
@@ -0,0 +1,81 @@
+const core = require('@actions/core');
+const fetch = require('node-fetch');
+const semverCoerce = require('semver/functions/coerce');
+const semverMajor = require('semver/functions/major');
+const semverMinor = require('semver/functions/minor');
+const semverGt = require('semver/functions/gt');
+
+async function getClusterVersions() {
+    const url = 'https://api.replicated.com/vendor/v3/cluster/versions';
+    const apiToken = core.getInput('replicated-api-token') || process.env.REPLICATED_API_TOKEN;
+    const headers = {
+        Authorization: apiToken
+    };
+
+    let clusterVersions = [];
+    try {
+        const response = await fetch(url, {
+            method: 'GET',
+            headers,
+        });
+
+        if (response.status === 200) {
+            const payload = await response.json();
+            clusterVersions = payload['cluster-versions'];
+        } else {
+            throw new Error(`Request failed with status code ${response.status}`);
+        }
+    } catch (error) {
+        console.error(`Error: ${error.message}`);
+        core.setFailed(error.message);
```

Comment:
is a `return` needed after this?

Reply:
good call. it might be ok, but won't hurt